tools/cldr-unittest/src/org/unicode/cldr/unittest/TestSupplementalInfo.java

```diff
@@ -561,9 +562,10 @@ public void TestDigitPluralCompleteness() {
             // % 100 = 1
             {"kw", "many", "00,000,0000"},  // n != 1 and n % 100 = 1,21,41,61,81
             {"kw", "zero", "0"},    // n = 0
+            {"fr", "many", ""},    // e is special
```

Comment:
This is the only real change. It says that fr doesn't have any real samples for 'many'